src/main/java/com/plaid/client/request/paymentinitiation/RecipientCreateRequest.java

```diff
@@ -12,10 +13,24 @@
   private String name;
   private String iban;
   private Address address;
+  private Bacs bacs;
 
   public RecipientCreateRequest(String name, String iban, Address address) {
     this.name = name;
     this.iban = iban;
     this.address = address;
   }
+
+  public RecipientCreateRequest(String name, String iban, Address address, Bacs bacs) {
```

Comment:
Is this how we handle parameter combinations for constructors in plaid-java elsewhere? If so, mind pointing me to an example? Or is this the first time doing something like this?

Reply:
Will update to do creation with helper methods